typescript/cli/src/context/strategies/chain/MultiChainResolver.ts

```diff
@@ -48,8 +46,6 @@ export class MultiChainResolver implements ChainResolver {
         return this.resolveWarpApplyChains(argv);
       case ChainSelectionMode.AGENT_KURTOSIS:
         return this.resolveAgentChains(argv);
-      case ChainSelectionMode.STRATEGY:
```

Comment:
Removing because having this here implies that we always expect `--strategy`, but it is not true. We want `--strategy` to be optional so we can default to jsonRpc strategy.